app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Dripbox</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<head>
+  <title>Dripbox</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
Diese gehören in den Head.

Reply:
Das habe ich nie verschoben hä????? Ich habe in meinen letzten Commit nachgeschaut und dort habe ich nix geändert. 
Das verstehe ich ja überhaupt nicht ??????????

Reply:
Ahja, ich sehe jetzt auch, dass die Änderung ohne Sinn ist, aber die beiden Zeilen sind im Header. Also, so erstmal OK. 
Hier ist der Commit von dir dazu https://github.com/Ktechen/Dripbox/commit/4b544548332b7b5e0e724ea0313ca11cbbb421c1#diff-f43fe075643e681b2c01c2f853bb0c4299d135b47fcbd4da96890d521c49e3eb

Hat mit Feature nichts zu tun, ist nur Einrückung der Zeilen.